Add more details on the release engineers role (#128)
Babylonpartners/ios-playbook: Cookbook/Technical-Documents/ReleaseProcess.md

    2. Escalating abnormal influx of bugs, so the release, as a whole, can be reassessed.
4. Making sure they can dedicate enough time for the release. In case this is not possible (due to other squad commitments), please inform the iOS chapter lead.

> **The objective is to ship the release build as soon as possible whilst maintaining the quality bar and addressing every bug raised by QA**. When dealing with a particularly complicated bug (one that would require a rather significant effort to address) the release engineer should speak with either Andreaa Papillon (Native Apps' Product Manager) or Lilla Szulyovszky (Native Apps' Delivery Manager) as every release is managed and signed-off by the Native Apps squad. The bug's corresponding feature should disabled altogether using its feature switch, if applicable. Such a bug would subsequently be handled by its respective squad.

Reviewer: `The bug's corresponding feature should disabled altogether using its feature switch`
->
`The bug's corresponding feature should be disabled altogether using its feature switch`

> **The objective is to ship the release build as soon as possible whilst maintaining the quality bar and addressing every bug raised by QA**. When dealing with a particularly complicated bug (one that would require a rather significant effort to address) the release engineer should speak with either Andreaa Papillon (Native Apps' Product Manager) or Lilla Szulyovszky (Native Apps' Delivery Manager) as every release is managed and signed-off by the Native Apps squad. The bug's corresponding feature should disabled altogether using its feature switch, if applicable. Such a bug would subsequently be handled by its respective squad.

Reviewer 2: ```suggestion
**The objective is to ship the release build as soon as possible whilst maintaining the quality bar and addressing every bug raised by QA**.
When dealing with a particularly complicated bug (one that would require a rather significant effort to address) the release engineer should speak with either Andreaa Papillon (Native Apps' Product Manager) or Lilla Szulyovszky (Native Apps' Delivery Manager) as every release is managed and signed-off by the Native Apps squad.
The bug's corresponding feature should be disabled altogether using its feature switch (if applicable). Such a bug would subsequently be handled by its respective squad.
```

> Release duties have priority over regular squad duties. Please inform your squad of your unavailability before starting to work in the release. It is a well known fact that twenty percent of your work hours overall are to be allocated towards making the release a reality.

Reviewer 2: ```suggestion
Release duties have priority over regular squad duties. Please inform your squad of your unavailability before starting to work in the release. As with every other week, twenty percent of your work hours overall are to be allocated towards making the release a reality.
```

> There are usually two release engineers working at any given time. It goes without saying that both engineers need to work together and that constant feedback is vital.

Reviewer 2: ```suggestion
There are usually two release engineers working at any given time. As both engineers have to work together, please be in constant communication with the other.
```

Author: @diegopetrucci I prefer my version to be honest. We always have to work together, only in this case even more so.
